crossbeam-deque/src/deque.rs

```diff
@@ -0,0 +1,1987 @@
+use std::cell::{Cell, UnsafeCell};
```

Comment:
This file is increasing the diff, but it really just moved the all implementation from lib.rs.